packages/config/config/devices/0x0234/zba7140.json

```diff
@@ -0,0 +1,279 @@
+{
+	"manufacturer": "Logic Group",
+	"manufacturerId": "0x0234",
+	"label": "ZBA7140",
+	"description": "Battery Wall Controller",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x0129"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic Report, Pushbutton 1 ",
```

Comment:
For consistency with other files, please make these "Button 1 (Basic Set)", "Button 1 (Multilevel Set)", etc. Carry this to other files.

Reply:
What would this be called?
	"label": "Binary Switch Report, Pushbutton 1",

Reply:
Button 1 (Binary Switch Report)